heudiconv/convert.py

```diff
@@ -1,3 +1,4 @@
+from filelock import SoftFileLock
```

Comment:
Note: I personally not familiar with `filelock` (used [fasteners](https://github.com/harlowja/fasteners) when needed), but [it](https://github.com/benediktschmitt/py-filelock) seems to be alive, so ok